package.json

```diff
@@ -32,6 +33,21 @@
     "@types/node": "^18.0.0",
     "@types/node-cron": "^3.0.1",
     "eslint": "^8.18.0",
+=======
```

Comment:
Same here

Reply:
@Juilzze resolved this issue in latest commit.